test/unit/vfocus/iterations.spec.js

```diff
@@ -0,0 +1,150 @@
+var path = require('path');
+var _ = require('lodash');
+var chai = require('chai');
+var expect = chai.expect;
+
+var h = require('virtual-hyperscript');
+var VText = require('vtree/vtext');
+var VFocus = require(path.resolve(process.cwd(), 'src/vfocus'));
+
+describe('VFocus - Iterations', function() {
+
+
+	describe('forEach', function() {
+		var forEachTree, forEachTreeFocus;
+		beforeEach(function() {
+			forEachTree = h(
+      	'div#0', [
+        	h('p#1', [
+          	h('b#2'),
+          	h('i#3')
+        	]),
+        	h('p#4')
+    	]);
+    
+    	forEachTreeFocus = new VFocus(forEachTree);
+		});
+
+
+		it('iterates through the right nodes in the right order', function(){
+			var nodesIds = [];
```

Comment:
The indentation is different than in the rest of the plugin (we use 2 spaces). You should be able to convert the whole file in Sublime.